files/common/usr/bin/delphix-startup-screen

```diff
@@ -390,5 +394,11 @@ def installer_main(stdscr):
 
 if __name__ == '__main__':
     logging.basicConfig(filename=log_file, level=logging.DEBUG)
-
+    signal.signal(signal.SIGTTOU, signal.SIG_IGN)
+    signal.signal(signal.SIGTSTP, signal.SIG_IGN)
+    signal.signal(signal.SIGHUP, signal.SIG_IGN)
+    signal.signal(signal.SIGINT, signal.SIG_IGN)
+    signal.signal(signal.SIGQUIT, signal.SIG_IGN)
```

Comment:
Can you comment on why we ignore those signals in particular?

Reply:
We are ignoring them so that you can't quit the login process when we exec it.

Reply:
Who would sent those signals normally? if someone presses enter to login to the console and then changes their mind and presses ctrl-C, what would happen before and after this change?

Reply:
Before if you press ctrl-C then it just prints on the screen. If we don't ignore the signal then the python process exits and restarts. So ignoring these signals allow the login behavior to stay the same.

Reply:
Got it, thanks, could you add this info as a comment in the code?